worlds/ror2/__init__.py

```diff
@@ -110,14 +110,14 @@ def set_rules(self) -> None:
         self.multiworld.completion_condition[self.player] = lambda state: state.has("Victory", self.player)
 
     def fill_slot_data(self):
-        return {
-            "itemPickupStep": self.multiworld.item_pickup_step[self.player].value,
-            "seed": "".join(self.multiworld.slot_seeds[self.player].choice(string.digits) for _ in range(16)),
-            "totalLocations": self.multiworld.total_locations[self.player].value,
-            "totalRevivals": self.multiworld.total_revivals[self.player].value,
-            "startWithDio": self.multiworld.start_with_revive[self.player].value,
-            "FinalStageDeath": self.multiworld.final_stage_death[self.player].value
-        }
+        option_results: Dict[str, Any] = {}
+        for option in ror2_options:
+            split_name = option.split(" ")
+            split_name = [name.title() for name in split_name]
+            split_name[0].lower()
```

Comment:
this needs to save back into split_name. 
split_name[0] = split_name[0].lower()